### DISCUSSION TITLE
Which 'Adult Content' label should be applied if specifics are unclear?

### DISCUSSION THREAD

**Discussion by qazmlp** [bluesky-social/atproto#2641] [How to do it?] [answered]
This is related to https://github.com/snarfed/bridgy-fed/issues/1100#issuecomment-2227783580.

While I don't work on any project directly, I'm curious what the current guidance is for interop with systems that don't have differentiated adult content labels to map to ATProto (and where a more specific label can't be determined because of resource constraints).

I assume that some label should be applied, so the question is "which?" between 'Suggestive', 'Nudity' and 'Porn'.
(Though, technically the 'sensitive' label on ActivityPub is used also for a lot of things that aren't covered by Bluesky's labels.)

**Comment by bnewbold** [comment, marked as answer]
I don't have a good answer for this today, and am not sure a good answer is possible. One system has granularity, the other doesn't. If one of the bsky labels was used as a "default", that would also change the de facto semantics of that label, which would not be good.

We are considering changes to the labels we use and apply, and can consider adding more generic labels to improve interop.

**Reply by qazmlp** [reply]
For what it's worth, Bridgy Fed is now/currently using "Graphic Media"/`graphic-media` as of https://github.com/snarfed/bridgy-fed/issues/1100#issuecomment-2228058295.

I don't know if this would lead to any concerns if used for adult content, though in those cases the original ActivityPub posts should often have a `summary` that clarifies it and is bridged in square brackets as extra line at the start of the post.

(Conversely, these self-labels also currently turn into matching `summary`s over there, but `summary` is fundamentally free-form and not machine-readable.)
